src/main/java/com/solvd/Main.java

```diff
@@ -363,6 +218,181 @@ public static void userActions(Scanner scanner, Client client) throws EntityNotF
+        System.out.print("2. Enter description: ");
+        String description = scanner.nextLine();
+
+        boolean badType = true;
+        do {
+            System.out.print("3. Enter Real Estate Type: ");
+            System.out.println("Choose type of Real Estate. Enter 1 or 2.\n" +
+                    "\t1. Apartament\n" +
+                    "\t2. Building");
+            String apartmentType = scanner.nextLine();
+            switch (apartmentType) {
+                case "1":
+                    realEstate.setRealEstateType(RealEstateType.APARTMENT);
+                    badType = false;
+                    break;
+                case "2":
+                    realEstate.setRealEstateType(RealEstateType.BUILDING);
+                    badType = false;
+                    break;
+                default:
+                    System.out.println("Invalid real estate type. Try again");
+            }
+        } while (badType);
+
+
+        System.out.print("4. Enter real estate metrics: ");
+        String metrics = scanner.nextLine();
+        System.out.print("5. Enter rooms: ");
+        String rooms = scanner.nextLine();
+
+        try {
+            realEstate.setPrice(parseDouble(price));
+            realEstate.setAvailable(true);
+            realEstate.setDescription(description);
+            realEstate.setMetrics(metrics);
+            realEstate.setRooms(parseInt(rooms));
+            realEstate.setSeller(client);
+            realEstate.setAddress(address);
+            REAL_ESTATE_SERVICE.create(realEstate, client.getId());
+            System.out.println("\n" + "Thanks! You've successfully created a real estate! ");
+        } catch (IllegalArgumentException | NullPointerException | EntityNotFoundException |
+                 LinkAlreadyExistsException e) {
+            System.out.println("\n" + e.getMessage());
+            System.out.println("Please try again.");
+        }
+    }
+
+    private static void viewRealEstateByType(Scanner scanner) {
+        System.out.println("Choose what type of Real Estate you are looking for (enter 1 or 2)\n" +
+                "\t1. Apartament\n" +
+                "\t2. Building");
+        RealEstateType realEstateType;
+        String typeChoice = scanner.nextLine();
+        switch (typeChoice) {
+            case "1":
+                realEstateType = RealEstateType.APARTMENT;
+                List<RealEstate> apartRealEstates = REAL_ESTATE_SERVICE.getAllByType(realEstateType);
+                for (RealEstate rlSt : apartRealEstates) {
+                    System.out.println(rlSt);
+                }
+                break;
+            case "2":
+                realEstateType = RealEstateType.BUILDING;
+                List<RealEstate> buildRealEstates = REAL_ESTATE_SERVICE.getAllByType(realEstateType);
+                for (RealEstate rlSt : buildRealEstates) {
+                    System.out.println(rlSt);
+                }
+                break;
```

Comment:
for-each loops duplication. Possibly you can extract them out of switch-case and then check `if (realEstates.isEmpty()) sout(NONE_MESSAGE) else for-each sout()`. And if so, you should add `return;` to default